__tests__/endpoints.test.js

```diff
@@ -100,3 +100,72 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+  it("PATCH /api/reviews/2 should update the voting count of that review", () => {
+    const newVotes = { inc_votes: 4 };
+    // const otherVotes = { inc_votes: -10 };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.votes).toBe(9);
+      });
+  });
+
+  it("PATCH /api/reviews/2 should update the voting count (negative) of that review", () => {
+    const otherVotes = { inc_votes: -10 };
+    return request(app)
+      .patch("/api/reviews/4")
+      .send(otherVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.votes).toBe(-3);
+      });
+  });
+
+  it("PATCH /api/reviews/223112 should respond with an error message and 404 error as the id does not exist", () => {
+    const otherVotes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/reviews/223112")
+      .send(otherVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid ID");
+      });
+  });
+
+  it("PATCH /api/reviews/five should respond with an error message and 404 error as the id is not an integer", () => {
+    const otherVotes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/reviews/five")
+      .send(otherVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Request");
+      });
+  });
+
+  it("PATCH /api/reviews/5 with number of votes in the wrong format should respond with an error message as number of votes should be integer", () => {
+    const otherVotes = { inc_votes: "one" };
+    return request(app)
+      .patch("/api/reviews/5")
+      .send(otherVotes)
+      .expect(404)
```

Comment:
this should also be a "400" - bad request, as it's also breaking the sql query

Reply:
Done